### DISCUSSION TITLE
Question: How to avoid tick overlap?

### DISCUSSION THREAD

**Discussion by 0lru** [epezent/implot#655] [Q&A]
Hi. Love the library. My biggest problem while using it is the tick spacing:
![image](https://user-images.githubusercontent.com/61606260/125618817-7424e2c5-93d8-4891-8cba-5327b25508ce.png)

I think the tick placing/spacing should not change when the plot- is at a different size (same data):
![image](https://user-images.githubusercontent.com/61606260/125619185-977edbfc-10dc-454f-adc4-d682c7139a53.png)

**Comment by epezent** [comment]
Thanks for the feedback. I just made a few changes to the algorithm in https://github.com/epezent/implot/commit/2dc2a4cfd5e3f0419b1f864cdaedba94ec8ef3a5.  Can you give the latest master a try and let me know if this improves your experience.

**Comment by 0lru** [comment]
definitely better:
![image](https://user-images.githubusercontent.com/61606260/125899482-f0f8eed2-055f-4ade-8178-c0c4f94fdada.png)

**Comment by 0lru** [comment]
Hm. This happens with -very- large numbers:
![image](https://user-images.githubusercontent.com/61606260/125899725-8d96a005-0d16-4f4f-ae82-60cf3c193fbd.png)

**Comment by epezent** [comment]
Good to know. I think we can solve this case with a second stage of pruning. Shouldn't be to hard to implement.
